cfme/tests/configure/test_visuals.py

```diff
@@ -92,3 +128,28 @@ def test_start_page(request, provider_init, start_page):
     login.login_admin()
     level = re.split(r"\/", start_page)
     assert menu.is_page_active(level[0].strip(), level[1].strip()), "Landing Page Failed"
+
+
+def test_infraprovider_noquads(request, provider_init, set_infra_provider_quad):
+    sel.force_navigate('infrastructure_providers')
+    assert visual.check_image_exists, "Image View Failed!"
+
+
+def test_host_noquads(request, provider_init, set_host_quad):
+    sel.force_navigate('infrastructure_hosts')
+    assert visual.check_image_exists, "Image View Failed!"
+
+
+def test_datastore_noquads(request, provider_init, set_datastore_quad):
```

Comment:
This entire suite should lose provider_init and begin to use the global setup_provider fixture in fixtures/provider.py